### PR TITLE
[14.0][FIX] connector_elasticsearch: Requests fail on >=7.14.0

### PR DESCRIPTION
**Pull request by rousseldenis**

### CODE REVIEW

**Comment by rousseldenis** [comment]
@sebastienbeau @simahawk 

Related to:

https://github.com/shopinvader/odoo-shopinvader/issues/1058

File: connector_elasticsearch/__manifest__.py [hunk 1]
```diff
@@ -22,6 +22,6 @@
         "views/se_menu.xml",
     ],
     "demo": ["demo/backend_demo.xml"],
-    "external_dependencies": {"python": ["elasticsearch>=7.0.0,<8.0.0", "requests"]},
+    "external_dependencies": {"python": ["elasticsearch>=7.0.0,<=7.13.4", "requests"]},
```
Comment (simahawk) [inline connector_elasticsearch/__manifest__.py:26]: maybe add a TODO as reminder to check?

**Review by simahawk — APPROVED** [review]

File: connector_elasticsearch/__manifest__.py (hunk 1 above)
Comment (rousseldenis) [reply connector_elasticsearch/__manifest__.py:26]: Done

**Comment by rousseldenis** [comment]
@simahawk If you can merge, we can go forward. Thanks

**Pull request merged**